includes/classes/payment.php

```diff
@@ -267,7 +267,7 @@ function after_order_create($zf_order_id) {
     if (!$GLOBALS[$this->selected_module]->enabled) return;
     $function = __FUNCTION__;
     if (!method_exists($GLOBALS[$this->selected_module], $function)) return;
-    return $GLOBALS[$this->selected_module]->after_order_create($function);
+    return $GLOBALS[$this->selected_module]->after_order_create($zf_order_id);

```

Comment:
In the spirit of @DrByte's original change, shouldn't this and the following change read
```php
   return $GLOBALS['this->selected_module]->$function($zf_order_id);
```

Reply:
Was just looking at that as I was closing tabs... Yes... Making second commit.